assets/pages/new-contributions-landing/contributionsLanding.scss

```diff
@@ -917,6 +917,10 @@ form {
   margin-bottom: $gu-v-spacing + 20px;
```

Comment:
weirdly there's a separate `.confirmation--backtothegu` just above, you could probably merge the two (not directly to do with your PR though)

Reply:
Merged.